### DISCUSSION TITLE
tutorials/performance/gpu_optimization

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#129] [User-contributed notes]
# tutorials/performance/gpu_optimization

Introduction: The demand for new graphics features and progress almost guarantees that you will encounter graphics bottlenecks. Some of these can be on the CPU side, for instance in calculations in...

https://docs.godotengine.org/en/stable/tutorials/performance/gpu_optimization.html

**Comment by Egalexandr** [comment]
If you're using Camera2D.zoom property, it could be a good decision to (re)import images with mipmap generation enabled.

**Comment by bar-ji** [comment]
The term "non-unshaded" is used in the fragment section. I think this causes unnecessary confusion over "shaded"

**Reply by Calinou** [reply]
Pull request opened:

- https://github.com/godotengine/godot-docs/pull/11800
